web/src/types/ga-gtag/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * npm module name:- "ga-gtag"
+ * version:-  "^1.0.1"
```

Comment:
The `:-` compound point is has been an [obsolete](https://en.wikipedia.org/wiki/Compound_point) punctuation combination for about a century; people who are alive just use `:` for this purpose.

`^1.0.1` is a [version range](https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004) (meaning `>=1.0.1 <2.0.0-0`), not a version.

We are using the current version, 1.2.0.